code/game/objects/items/devices/PDA/PDA_types.dm

```diff
@@ -10,7 +10,7 @@
 
 /obj/item/pda/clown/ComponentInitialize()
 	. = ..()
-	AddComponent(/datum/component/slippery, 7 SECONDS, NO_SLIP_WHEN_WALKING, CALLBACK(src, .proc/AfterSlip), 5 SECONDS)
+	AddComponent(/datum/component/slippery/clowning, 120, NO_SLIP_WHEN_WALKING, CALLBACK(src, .proc/AfterSlip), slot_whitelist = list(ITEM_SLOT_ID, ITEM_SLOT_BELT))
```

Comment:
Slip time appears to be about 17 seconds in my tests.
Reduce it back down to it's original 7 SECONDS value otherwise it's going to be far too powerful.